cmd/cmd_controller.js

```diff
@@ -56,17 +56,24 @@ class EmbarkController {
     self.context = options.context || [constants.contexts.run, constants.contexts.build];
     let Dashboard = require('./dashboard/dashboard.js');
 
-    let webServerConfig = {
-      enabled: options.runWebserver
-    };
+    const webServerConfig = {};
 
-    if (options.serverHost) {
+    if (options.runWebserver != null) {
```

Comment:
Is there a reason of using `!=` over `!==`

Reply:
`!= null` will only match `null` or `undefined`.